api/src/environment.js

```diff
@@ -48,16 +48,9 @@ module.exports.setDeployInfo = newDeployInfo => {
 };
 
 module.exports.getDeployInfo = () => deployInfo;
-
-module.exports.getExtractedResourcesPath = () => {
-  let destination = MEDIC_API_RESOURCE_PATH;
-  if (!destination) {
-    const isProduction = NODE_ENV === 'production';
-    const defaultLocation = path.join(__dirname, '..', 'extracted-resources');
-    destination = isProduction ? '/tmp/extracted-resources' : defaultLocation;
-  }
-
-  return path.resolve(destination);
-};
-
+module.exports.getBuildPath = () => path.join(__dirname, '..', 'build');
+module.exports.getStaticPath = () => path.join(module.exports.getBuildPath(), 'static');
+module.exports.getWebappPath = () => path.join(module.exports.getStaticPath(), 'webapp');
+module.exports.getDefaultDocsPath = () => path.join(module.exports.getBuildPath(), 'default-docs');
+module.exports.getResourcesPath = () => path.join(__dirname, '..', 'resources');
```

Comment:
Can these be exported constants instead of functions?

Reply:
Done!